e2e/test/TwinE2ETests.cs

```diff
@@ -750,10 +750,7 @@ static public void ClassCleanup()
                 Assert.IsTrue(serviceTwin.Properties.Reported.Contains(propName1));
                 String value1 = serviceTwin.Properties.Reported[propName1].ToString();
 
-                // TODO: #243 
-                // If service team fixed the Issue #243 the following Assert.AreNotEqual should fail 
-                // and need to be changed to Assert.AreEqual
-                Assert.AreNotEqual(value1, propEmptyValue);
+                Assert.AreEqual(value1, propEmptyValue);
```

Comment:
@abhipsaMisra @zolvarga The tests are going to be in an odd state as our service gets upgraded.
My suggestion is to disable the tests against a GitHub issue (maybe the original one) until things get stable. At that point we can simply enable the test validating the new service behavior and close the bug.